react/exercise/Q1-1/index.tsx

```diff
@@ -0,0 +1,31 @@
+// Q1-1
+// * 任意の赤色の物の名前, 緑色の物の名前, 青色の物の名前を JSX 内に追加
+//  * それぞれが赤色, 青色, 緑色となるようにスタイルを指定
+//  * 並びや位置関係、厳密に物の色が正しいかどうかは不問
+//
+import { FC } from "react";
+import { createRoot } from "react-dom/client";
+import "./style.css";
+
+const ChildComponent: FC = () => {
+  return (
+    <div>
+      {/*
+        // 赤色の物の名前を入力し、赤色ものは赤色に
+        // 緑色の物の名前を入力し、緑色ものは赤色に
+        // 青色の物の名前を入力し、青色ものは赤色に
```

Comment:
```suggestion
        // 赤色の物の名前を入力し、赤色で表示
        // 緑色の物の名前を入力し、緑色で表示
        // 青色の物の名前を入力し、青色で表示
```
すみません、スペースもですが、意図としては文言の方の指摘でした。

Reply:
@aNobuhisaMasaki 
すいません、見落としてました。
efe634ee293e58405354fe41a7bba46e4bbeacba